### DISCUSSION TITLE
Reporting data provenance

### DISCUSSION THREAD

**Discussion by pandu-supriyono** [sProDev/kodepos#42] [Q&A]
Assalamualaikum warahmatullahi wabarakatuh

This thread is a continuation of #40.

[https://kodepos.posindonesia.co.id/](https://kodepos.posindonesia.co.id/) is a service of PT Pos Indonesia, only it seems that their FAQ page returns a 404.

Unfortunately, I cannot find further resources that speak about this source, its reliability in terms of who is responsible for keeping the data up-to-date nor an open data policy.

If you want, I am willing to reach out to them by phone or mail to ask for further clarification.

**Comment by sooluh** [comment]
وعليكم السلام ورحمةاللّٰه وبركاته

FYI, Indonesian citizens might already trust the data provided by [posindonesia.co.id](https://www.posindonesia.co.id/en), and I can even say they have to agree.

POS Indonesia is a state-owned company (we call it BUMN) that has been in the postal business for a long time. You can check out more information on their Wikipedia page: [Pos Indonesia](https://en.wikipedia.org/wiki/Pos_Indonesia).

Postal codes in Indonesia have a codification that you can learn about here: [Postal codes in Indonesia](https://en.wikipedia.org/wiki/Postal_codes_in_Indonesia). You can also visit the official website of POS Indonesia at [POS Indonesia Official Website](https://www.posindonesia.co.id/en).

In the footer of the official website, there is contact information where you might be able to request clarifications. From my perspective and that of Indonesian citizens, we trust the existing data. Or maybe it shouldn't be this way?

**Comment by pandu-supriyono** [comment]
Pos Indonesia is a trusted authority, so I understand the valid assumption that their data is reliable. However, for clients who need high accountability, like those with ISO-9001 certification, there are some important considerations.

Clients will need to know how often the data is updated and see a clear commitment to good and transparent data governance. They also need clarity on the terms of use and licensing. Without this information, it's hard to confidently validate the use of the data in a prototype.

There are models, like the [5-star open data model](https://5stardata.info/en/), that help evaluate how open and well-governed a data source is. It would be helpful to see how Pos Indonesia measures up against these standards. Perhaps we might even be able to get access to a machine-readable version of their data which eliminates the need to manually grab the data from their .PDF file.
